src/App.js

```diff
@@ -2,7 +2,7 @@ import { Routes, Route } from "react-router-dom";
 import "./App.css";
 import { Toaster } from "react-hot-toast";
 
-import { Navbar } from "./Components/Index";
+import { Navbar, RequiresAuth, Sidebar } from "./Components/Index";
```

Comment:
No need to mention index after "./Components";
`import { Navbar, RequiresAuth, Sidebar } from "./Components/";`
This will also work out